### ISSUE TITLE
Minor change of magic-sealed-data format

### ISSUE THREAD

**Issue by taw** [dr4fters/dr4ft#983]
This needed to be done to support booster variants.

Fields `name` and `code` were set name and code, as there was 1-to-1 correspondence between sets and booster types.

Now they're used for full name of specific booster variant, and code (which is always going to be `(set_code)-(variant_code)`).

I added `set_code` and `set_name` that give code and name of the set.

For now they match exactly, except for one new variant booster:

 ```
{
    "name": "Alara Premium Foil Booster",
    "code": "ala-premium",
    "set_code": "ala",
    "set_name": "Shards of Alara",
    ...
    (the rest like before)
}
```

WotC created a lot of variant boosters in last few years. I think this format will be flexible enough I won't need any more changes.

I hope it doesn't create too much work for you guys.

**Comment by HerveH44** [comment]
Hi @taw ,

Thanks for the update. I don't think it creates any work from us if the "code" for basic booster still equals to the set code (for example the object with "ala" code refers to the normal booster sheets).

**Comment by taw** [comment]
Yeah, for basic boosters, `code` and `set_code` will be the same thing, as will `name` and `set_name`.

**Comment by taw** [comment]
@ZeldaZach wanted some other minor changes so it can get integrated into mtgjson. Can you two discuss what works for you best? (as far as I know there are no other users for than json yet)

**Comment by taw** [comment]
I added some documentation to https://github.com/taw/magic-sealed-data as README.md

I hope this is enough. If you have any requests, just let me know (creating issue in that repo, or whichever other way works for you)
